pkg/version/fixbuild.go

```diff
@@ -9,14 +9,27 @@ func init() {
 }
 
 func buildInfoFixBuild(v *Version) {
+	if v.Build != "$Id$" {
```

Comment:
catches when not using git hash (i.e. linker field like `go build -ldflags="-X github.com/go-delve/delve/pkg/version.BuildVersion=your-custom-build" -o dlv ./cmd/dlv`); without it'll fall back to git revision hash.